### PR TITLE
Update to newest Neptune, AppMan and QtWebEnigne

### PR DESCRIPTION
**Pull request by rpannek**
Some things in the service file needed to be reworked to work with the
newest version of appman and neptune.

Signed-off-by: Richard Pannek <richard.pannek@pelagicore.com>

### CODE REVIEW

File: recipes-qt/automotive/qtapplicationmanager_git.bbappend [hunk 1]
```diff
@@ -2,4 +2,4 @@
 # provides "geniviextras" for Qt which QtAM uses for DLT-logging
 DEPENDS += "qtivi"
 
-SRCREV = "05a6ca178d7d95128a2e0053d06db56a844e4696"
+#SRCREV = "05a6ca178d7d95128a2e0053d06db56a844e4696"
```
Comment (breakreturn) [inline recipes-qt/automotive/qtapplicationmanager_git.bbappend]: What's the purpose of leaving this in here? Do we expect to uncomment it soon? Should there be a comment in that case?

Comment (GordanM) [reply recipes-qt/automotive/qtapplicationmanager_git.bbappend]: +1

Comment (rpannek) [reply recipes-qt/automotive/qtapplicationmanager_git.bbappend]: Ah damn, that should have been changed to the real srcrev instead, I will fix that.

Comment (rpannek) [reply recipes-qt/automotive/qtapplicationmanager_git.bbappend]: fixed

File: recipes-dev-platform/images/genivi-dev-platform.bbappend [hunk 2]
```diff
@@ -1,5 +1,11 @@
-# Don't ship the standard GDB UI
-IMAGE_INSTALL_remove = "packagegroup-gdp-hmi"
+# Don't ship the standard GDB, the browser and weston
+IMAGE_INSTALL_remove = " \
+    packagegroup-gdp-hmi \
+    packagegroup-gdp-browser \
+    wayland-ivi-extension \
+    packagegroup-abstract-component-p1 \
+    weston \
```
Comment (GordanM) [inline recipes-dev-platform/images/genivi-dev-platform.bbappend:7]: Nice catch with this removals!

**Review by GordanM — COMMENTED** [review]
Looks good to me, follows the recent changes to neptune startup nicely.

**Review by GordanM — APPROVED** [review]

File: recipes-dev-platform/images/genivi-dev-platform.bbappend (hunk 2 above)
Comment (rpannek) [reply recipes-dev-platform/images/genivi-dev-platform.bbappend:7]: Yeah, weston would sometimes start befor AppMan and then AppMan wouldn't be able to start.

**Review by breakreturn — APPROVED** [review]

**Pull request merged**